content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/_index.md

```diff
@@ -13,23 +13,18 @@ Navigate to the **Settings** page in the Cobalt app.
 If your user role allows, you can:
 
 - Edit the [organization profile](#edit-the-organization-profile)
-- Configure [advanced settings](#manage-identity-and-access-settings): two-factor authentication and SAML-based single sign-on
+- Configure [security settings](#manage-security-settings)
```

Comment:
we still call this tab Identity & Access https://www.loom.com/i/e384040fad2e44bcab29868ecb585984

Reply:
Renamed across all files.